crates/debugger_ui/src/variable_list.rs

```diff
@@ -451,7 +451,14 @@ impl VariableList {
                 );
             }
 
-            let result = try_join_all(tasks).await?;
+            let tasks_results = futures::future::join_all(tasks).await;
+            let result = tasks_results
```

Comment:
I think we can make this `result.into_iter().filter_map(|result| result.ok())` could you also make clippy pass? I think we have one unused import for **try_join_all**.

Reply:
That indeed looks better and I also removed the unused import.